tests/docs/autodoc/docs/index.rst

```diff
@@ -0,0 +1,35 @@
+.. Example documentation master file, created by
+   sphinx-quickstart on Sat Sep  7 17:17:38 2013.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to an advanced documentation example integrating with autodoc
+=====================================================================
+
+Contents
+--------
+
+.. toctree::
+   :maxdepth: 2
+
+   prod_code
+   test_code
+
+Traceability Matrix
+-------------------
+
+Bi-directional traceability (forward = requirements -> test cases,
+backward = test cases -> requirements) of the software requirements and the test
+cases provide information about missing requirements and/or missing test cases.
+
+.. item-matrix:: title
+   :source: SW_REQ_
+   :target: SW_TEST_
+   :type: validates validated_by
```

Comment:
There's no need to indicate `validates` here. Just setting type `validated_by` is enough, as the `item-matrix` directive automatically checks for the reverse relationship (test case -- validates --> requirement).

In other words, two items, source S and target T, are related with relationship R if:
```
.. item:: S
   :R: T
```
or
```
.. item:: T
   :V: S
```
being V the inverse relationship of R